AudioKitSynthOne/AudioKitSynthOne.entitlements

```diff
@@ -6,17 +6,17 @@
 	<string>development</string>
 	<key>com.apple.developer.icloud-container-identifiers</key>
 	<array>
-		<string>iCloud.com.audiokitpro.AudioKitSynthOne</string>
-		<string>iCloud.com.audiokitpro.AudioKitSynthOne.Synth</string>
+		<string>iCloud.com.audiokitpro.AudioKitSynthOne.staque</string>
```

Comment:
Please exclude this file from the PR since it's just for your own benefit.

Reply:
Can I just delete the file from the PR or do I have to start a new one?

Reply:
I think you might be able to remove it from the PR in GitHub though I have never tried that myself.